brd-ios/breadwallet/src/Redesign/Core/CLEANerSwift/Controllers/VIPTableViewController.swift

```diff
@@ -63,42 +63,57 @@ class VIPTableViewController<C: CoordinatableRoutes,
     override func setupSubviews() {
         super.setupSubviews()
         
-        topInsetValue = sceneLeftAlignedTitle == nil ? 0 : (Margins.large.rawValue * 2) + 28
-        
-        view.addSubview(leftAlignedTitleLabel)
-        leftAlignedTitleLabel.snp.makeConstraints { make in
-            make.top.equalTo(view.safeAreaLayoutGuide.snp.top)
-            make.height.equalTo(topInsetValue)
-            make.leading.trailing.equalToSuperview().inset(Margins.large.rawValue)
-        }
+        topInsetValue = sceneLeftAlignedTitle == nil ? 0 : Margins.extraHuge.rawValue + 28
         
         view.addSubview(tableView)
         tableView.snp.makeConstraints { make in
             make.edges.equalToSuperview()
         }
         
+        tableView.addSubview(leftAlignedTitleLabel)
+        leftAlignedTitleLabel.snp.makeConstraints { make in
+            make.bottom.equalTo(tableView.snp.top)
+            make.height.equalTo(topInsetValue)
+            make.leading.trailing.equalToSuperview().inset(Margins.large.rawValue)
+        }
+        
         tableView.contentInset.top = topInsetValue
     }
     
     func setRoundedShadowBackground() {
+        let small = Margins.small.rawValue
```

Comment:
why not use a WrapperView and embed tableView in it.. and then just set sadow/corners on that? 
this seems.. a bit 2much for a VC